Reportedly/Modules/Authentication/LoginEmail/LoginEmailRouter.swift

```diff
@@ -32,4 +34,9 @@ final class LoginEmailRouter: LoginEmailRouterInput {
             self.window = window
         }
     }
+
+    func showSignupScreen() {
+        let module = SignupModule()
+        viewController?.navigationController?.pushViewController(module.view, animated: true)
```

Comment:
We use `pushViewController `, i think it will make more sense if we update `pushSignupScreen` instead of `showSignupScreen `. What do you think? 😅

Reply:
yup that makes sense to me, will update 👍